data/json/items/tools.json

```diff
@@ -74,6 +74,21 @@
       "type": "transform"
     }
   },
+  {
+    "id": "hand_pump",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "blue",
+    "name": "hand pump",
+    "name_plural": "hand pumps",
```

Comment:
Redundant.

Reply:
The plural name is redundant? it seems to fit the same place as the other items and their plural names

Reply:
The `s` is automatically appended if a plural name is missing, so there is no point having a plural name specify that.